src/controllers/userController.js

```diff
@@ -11,6 +13,30 @@ const signup = catchAsync(async (req, res) => {
   res.status(201).json({ message: "SIGNUP_SUCCESS" });
 });
 
+
+const signin = catchAsync(async (req, res) => {
+	const { email, password } = req.body;
+	if (!email || !password) detectError("NO INPUT", 404);
+
+	const accessToken = await userService.signIn(email, password);
+
+	res.status(201).json({ accessToken: accessToken });
+});
+
+
+const oAuth = catchAsync(async (req, res) => {
+	const { code } = req.query;
+
+	if (!code) detectError("NO AUTH CODE", 404);
```

Comment:
```suggestion
	if (!code) detectError("NO AUTH CODE", );
```
404는 보통 요청의 주소가 잘못된 경우를 의미합니다. 다른 status code를 사용해 주세요!